src/pages/EditMonitor.vue

```diff
@@ -385,6 +388,23 @@
                                 </div>
                             </template>
 
+                            <template v-if="monitor.type === 'zabbix-trigger'">
+                                <div class="my-3">
+                                    <label for="zabbix_instance_url" class="form-label">{{ $t("zabbixInstanceUrl") }}</label>
+                                    <input id="zabbix_instance_url" v-model="monitor.zabbixInstanceUrl" type="text" class="form-control" required />
+                                </div>
+
+                                <div class="my-3">
+                                    <label for="zabbix_trigger_id" class="form-label">{{ $t("zabbixTriggerId") }}</label>
+                                    <input id="zabbix_trigger_id" v-model="monitor.zabbixTriggerId" type="text" class="form-control" required />
+                                </div>
+
+                                <div class="my-3">
+                                    <label for="zabbix_auth_token" class="form-label">{{ $t("zabbixAuthToken") }}</label>
+                                    <input id="zabbix_auth_token" v-model="monitor.zabbixAuthToken" type="password" class="form-control" required />
```

Comment:
I have never used zabbix.
Is it nessesary to add a helptext how to find the auth token?